api-specificatie/zrc/openapi.yaml

```diff
@@ -1,83 +1,504 @@
               schema:
-                $ref: '#/components/schemas/Fout'
+                $ref: '#/components/schemas/ZaakObject'
+      tags:
+        - zaakobjecten
+    parameters:
+      - name: id
+        in: path
+        description: A unique integer value identifying this zaakobject.
+        required: true
+        schema:
+          type: integer
+  /zaken:
+    post:
+      operationId: zaak_create
+      description: >-
+        Maak een ZAAK aan.
+
+
+        Indien geen zaakidentificatie gegeven is, dan wordt deze automatisch
+        gegenereerd.
+      responses:
+        '201':
+          description: ''
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Zaak'
+      tags:
+        - zaken
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Zaak'
+        required: true
     parameters: []
+  '/zaken/{id}':
+    get:
+      operationId: zaak_read
+      description: Opvragen en bewerken van ZAAKen.
+      responses:
+        '200':
+          description: ''
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Zaak'
+      tags:
+        - zaken
+    parameters:
+      - name: id
+        in: path
+        description: A unique integer value identifying this zaak.
+        required: true
+        schema:
+          type: integer
+  '/zaken/{zaak_pk}/zaakeigenschappen':
+    get:
+      operationId: zaakeigenschap_list
+      description: Geef een collectie van eigenschappen behorend bij een ZAAK.
+      responses:
+        '200':
+          description: ''
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ZaakEigenschap'
+      tags:
+        - zaken
+    post:
+      operationId: zaakeigenschap_create
+      description: Registreer een eigenschap van een ZAAK.
+      responses:
+        '201':
+          description: ''
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ZaakEigenschap'
+      tags:
+        - zaken
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ZaakEigenschap'
+        required: true
+    parameters:
+      - name: zaak_pk
+        in: path
+        required: true
+        schema:
+          type: string
+  '/zaken/{zaak_pk}/zaakeigenschappen/{id}':
+    get:
+      operationId: zaakeigenschap_read
+      description: Geef de details van ZaakEigenschap voor een ZAAK.
+      responses:
+        '200':
+          description: ''
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ZaakEigenschap'
+      tags:
+        - zaken
+    parameters:
+      - name: id
```

Comment:
https://github.com/VNG-Realisatie/gemma-zaken/issues/208

Reply:
@michielverhoef ik heb dit net met Joeri besproken, en wil dit deze even parkeren zodat #208 opgepakt kan worden in een volgende sprint. Voor nu zou dit dus niet in de weg mogen zitten binnen dit pull request, aangezien we met linked data werken.

De enige aanpassing die ik moet doen is bronorganisatie als veld toevoegen aangezien dit een verplicht veld is. Dit zou ik echter in een losse pull request doen omdat het tegelijk #39 en #52 betreft en voor de functionaliteit van deze PR niet uitmaakt. Als je verder geen opmerkingen hebt, kan je deze PR dan approven zodat die gemerged kan worden?

Reply:
Ok, wat mij betreft kunnen we dan door.